helm/templates/acquire/deployment.yaml

```diff
@@ -40,6 +40,14 @@ spec:
               containerPort: 9568
               protocol: TCP
           env:
+            - name: ACQUIRE_DB_HOST
+              value: {{ template "postgres.fullname" . }}
+            - name: ACQUIRE_DB_NAME
+              value: {{ .Values.presto.view_state_apps.acquire.db.name }}
+            - name: ACQUIRE_DB_USER
+              value: {{ .Values.presto.view_state_apps.acquire.db.user | required "app db.user must be set" }}
+            - name: ACQUIRE_DB_PASSWORD
+              value: {{ .Values.presto.view_state_apps.acquire.db.password | required "app db.password must be set" }}
```

Comment:
Are we referencing these anywhere in code?

Reply:
Yes, in the releases.exs file to configure the Postgres driver connection.